proc-macros/Cargo.toml

```diff
@@ -17,6 +17,7 @@ proc-macro2 = "1.0"
 quote = "1.0"
 syn = { version = "1.0", default-features = false, features = ["extra-traits", "full", "visit", "parsing"] }
 proc-macro-crate = "1"
+convert_case = "0.6.0"
```

Comment:
Another alterantive would be [heck](https://github.com/withoutboats/heck) that we use in subxt. It has a bit more traction (git stats), No strong opinions on those, they both look fine.

Reply:
heck seems more widely used let's switch to it.